pom.xml

```diff
@@ -26,6 +26,16 @@
             <version>7.6.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.8.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.8.2</version>
+        </dependency>
 
     </dependencies>
 
```

Comment:
Please change maven.compiler.source and maven.compiler.target to "17"

Reply:
Thanks, was changed to 17